### ISSUE TITLE
"tagChildrenIndent" is not applied to script tags

### ISSUE THREAD

**Issue by DrRataplan** [yeonjuan/html-eslint#310]
html-eslint is Awesome! But I'm running into a small issue. When I use the '@html-eslint/indent.tagChildrenIndent' option to remove indents for the `html` tag, it forces a `script` tag that's a direct child of the html tag to still be indented.

Input:
```html
<!DOCTYPE html>
<html>
<head>
</head>
<body>
<div>
  <li> foo </li>
</div>
</body>
<script src="yadayada"></script>
</html>
```
```json
{
  "rules": {
    "@html-eslint/indent": [
      "error",
      2,
      {
        "tagChildrenIndent": {
          "html": 0,
          "body": 0
        }
      }
    ]
  }
}
```

Expected: document passes linting

Actual: error on line 10, the script tag is indented with two spaces.

Playground: https://html-eslint.org/playground#eyJjb2RlIjoiPCFET0NUWVBFIGh0bWw+XG48aHRtbD5cbjxoZWFkPlxuPC9oZWFkPlxuPGJvZHk+XG48ZGl2PlxuICA8bGk+IGZvbyA8L2xpPlxuPC9kaXY+XG48L2JvZHk+XG48c2NyaXB0IHNyYz1cInlhZGF5YWRhXCI+PC9zY3JpcHQ+XG48L2h0bWw+XG4iLCJjb25maWciOnsicnVsZXMiOnsiQGh0bWwtZXNsaW50L2luZGVudCI6WyJlcnJvciIsMix7InRhZ0NoaWxkcmVuSW5kZW50Ijp7Imh0bWwiOjAsImJvZHkiOjB9fV19fSwibGFuZ3VhZ2UiOiJodG1sIn0=

**Comment by yeonjuan** [comment]
@DrRataplan Thank you for your report. It's definitely a bug.

**Comment by DrRataplan** [comment]
Cool! Thanks!
